loadtest/store/memstore/store.go

```diff
@@ -315,3 +316,42 @@ func (s *MemStore) Roles() ([]model.Role, error) {
 	}
 	return roles, nil
 }
+
+// For all the random utilities we flatten the map to a slice
+// and then take a random index from the slice.
+// We do this, because even though the map iteration order
+// is "undefined" in the spec, practically it returns the
+// same set of values always.
+
+// RandomChannel returns a random channel for a user.
+func (s *MemStore) RandomChannel() (model.Channel, error) {
```

Comment:
I suggest we either add a `teamId` parameter or we add a function to pick a channel randomly but with the specified `teamId`.

Reply:
Yes, I was thinking about that too. Will update.